### ISSUE TITLE
CSP style_src unsafe_inline needed for development

### ISSUE THREAD

**Issue by KieranP** [ElMassimo/vite_ruby#168]
On installation, several lines are added to content_security_policy.rb. Based on my testing, an additional one is required to support inline styles when css is imported via JS:

```
policy.style_src *policy.style_src, :unsafe_inline if Rails.env.development?
```

**Comment by ElMassimo** [comment]
CSP configuration takes place outside of this library, the rules are provided as a starting point (and they are commented out).

PRs to update the examples are welcome!

**Comment by KieranP** [comment]
@ElMassimo Have created a PR, #169  :-)

**Comment by ElMassimo** [comment]
Closed in #169
